### ISSUE TITLE
The interactive examples didn't have width and height CSS properties, which make them invisible in other editor environment.

### ISSUE THREAD

**Issue by kuthathwe** [mdn/css-examples#154]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders

### What specific section or headline is this issue about?

Backgrounds and Borders

### What information was incorrect, unhelpful, or incomplete?

CSS properties of image elements should be placed in the interactive code. I tried to copy the code on sublime but didn't see the image, until I figured out that it needs width and height to be displayed.

### What did you expect to see?

Full CSS properties 

### Do you have anything more you want to share?

Everything on the MDN is fantastic, I enjoyed learning on MDN platform 💖

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/css/building_blocks/backgrounds_and_borders`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/building_blocks/backgrounds_and_borders/index.md
* Last commit: https://github.com/mdn/content/commit/acfe8c9f1f4145f77653a2bc64a9744b001358dc
* Document last modified: 2023-07-07T07:19:19.000Z

</details>

**Comment by caugner** [comment]
This is a limitation of the interactive editor we use for CSS examples.

The "Background images" example has two `<style>` elements:

- The first one is hidden:
   https://github.com/mdn/css-examples/blob/2d33a1c5aede31cc0e15415bce708950de582675/learn/backgrounds-borders/background-image.html#L7-L23
- The second is what the editor shows you:
  https://github.com/mdn/css-examples/blob/2d33a1c5aede31cc0e15415bce708950de582675/learn/backgrounds-borders/background-image.html#L25-L33

It would make sense if the user could see the non-editable portion of the CSS.

**Comment by iprinceroyy** [comment]
Hey @caugner, I've done a brief overview of the mentioned HTML file and linked JS and CSS files. I am ready to work on this issue. Is there any specific design or do we need to do it ourselves? Could you give me more context?

**Comment by caugner** [comment]
Yes, it looks like this CSS Example is no longer used in content, see [this GitHub search](https://github.com/search?q=repo%3Amdn%2Fcontent+%2Fcss-examples%2F+%2Fbackground-image%2F&type=code).
